docs/_sass/getnighthawk.scss

```diff
@@ -6226,6 +6226,14 @@ margin: 0;
   .des-para {
     font-size: 14px;
   }
+  .des-para1 {
```

Comment:
What is this being used for🤔? don't see this class called anywhere

Reply:
https://github.com/layer5io/getnighthawk/blob/master/docs/index.html#L192 looks like it is being used at this line @warunicorn19

Reply:
Ohhh okay😅